src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -631,25 +749,46 @@ else if ( NoteLine.contains(CuDisbursementVoucherConstants.DV_EXTRACT_NOTE_PREFI
                                     if (NoteLine.substring(0,2).contains(CuDisbursementVoucherConstants.DV_EXTRACT_TYPED_NOTE_PREFIX_IDENTIFIER)) {
                                         // Trim the first two characters from the note and assign it as the first user typed note line
                                         NoteLine = NoteLine.substring(2);
-                                        FirstNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
                                         
+                                        //Trim initialization data of any leading or trailing spaces for very first note line. 
+                                        //Subsequent methods called to build note lines deals with this internally for remaing note lines.
+                                        FirstNoteAfterAddressInfo = stripTrailingSpace(stripLeadingSpace(FirstNoteAfterAddressInfo));
+                                        
+                                        SecondNoteAfterAddressInfo = obtainNoteLineSectionExceedingCheckStubLine(NoteLine, FirstNoteAfterAddressInfo);
+                                        
+                                        FirstNoteAfterAddressInfo = (StringUtils.equals(FirstNoteAfterAddressInfo, KFSConstants.BLANK_SPACE) |
+                                                StringUtils.equals(FirstNoteAfterAddressInfo, KFSConstants.EMPTY_STRING))
```

Comment:
would suggest using a StringUtils.isBlank() check for simplicity. The same goes for the related code for the second-note and third-note handling.

Reply:
Adjusted here and in unit test so code matches.